eng/code-quality-reports/src/main/resources/revapi/revapi.json

```diff
@@ -140,7 +140,7 @@
         {
           "regex": true,
           "code": "java.class.externalClassExposedInAPI",
-          "new": "(class|interface|enum) com.azure.spring.(messaging|service|eventhubs|servicebus|integration|storage|cloud.(core|autoconfigure|stream.binder.(servicebus|eventhubs)|actuate)|resourcemanager)..*",
+          "new": "(class|interface|enum) com.azure.spring.(messaging|eventhubs|servicebus|integration|storage|cloud.(core|service|autoconfigure|stream.binder.(servicebus|eventhubs)|actuator|resourcemanager))..*",
```

Comment:
shouldn't this be messaging.eventhub and messaging.servicebus?

Reply:
Will fix in this PR https://github.com/Azure/azure-sdk-for-java/pull/27300

Reply:
@moarychan, we'd better user another pr  instead of using the native one, for we're not sure when will the native be able to be merged.